apps/src/util/experiments.js

```diff
@@ -121,6 +122,8 @@ experiments.isEnabled = function(key) {
       window.appOptions.experiments &&
       window.appOptions.experiments.includes(key)
     );
+  // Check DCDO to see if this experiment is enabled.
+  enabled = enabled || DCDO.get(key, false);
```

Comment:
this is minor, but to honor the contract of the isEnabled method, this should be 
```suggestion
  enabled = enabled || !!DCDO.get(key, false);
```

Reply:
Since the second parameter to `DCDO.get` is the default, can we simplify to this?
```
enabled = DCDO.get(key, enabled)
```
I guess it depends whether we want the DCDO key to be able to override the value of `enabled` when it's `true`...

Reply:
> I guess it depends whether we want the DCDO key to be able to override the value of enabled when it's true...
Right! I do want the targeted experiments like User flags or cookie flags to take priority over global experiments. That way we can test turning an experiment off on an individual account before turning off the feature globally. Therefore, I'm going to leave the code as it is.